### PR TITLE
Added a fallback for arm Cpus

### LINKED ISSUE #97: [BUG] uno-check creates a x86_64 android emulator for M1
When I run the uno-check tooling in an M1 device, and I don't have an Android Emulator, it creates a `x86_64` device which is incompatible with M1.

1. Delete all androids emulator that you may have
2. Run the uno-check and type `y` when asked to install the Android Emulator
3. See the emulator installed

Before run the uno-check:
Screen Shot 2022-09-28 at 13 57 19

When uno-check is done:
Screen Shot 2022-09-28 at 13 57 50

I would like that uno-check knows that I'm running on a arm device and install the correct emulator for me. (hope that's possible)

### PR DESCRIPTION
**Pull request by pictos**
This PR makes sure that `uno-check` will install the Android Emulator for arm64.

-Fixes #97

### CODE REVIEW

File: UnoCheck/Checkups/AndroidEmulatorCheckup.cs [hunk 1]
```diff
@@ -114,8 +114,15 @@ public override Task<DiagnosticResult> Examine(SharedState history)
...
 
-								var sdkPackage = installedPackages.FirstOrDefault(p => p.Path.Equals(me.SdkId, StringComparison.OrdinalIgnoreCase));
-
+								var sdkPackage = installedPackages.FirstOrDefault(p =>
+								{
+									// This will be false if the proccess runs on Rosetta emulation
+									// and will install the wrong emulator (x86_64)
+									// https://github.com/dotnet/runtime/issues/42130
+									return Util.IsArm64
+									? p.Path.Contains("arm", StringComparison.OrdinalIgnoreCase)
```
Comment (jeromelaban) [inline UnoCheck/Checkups/AndroidEmulatorCheckup.cs:123]: Can you add in the comment here an example of the `p.Path` value? I wonder if we're talking about emulator architecture path or the SDK itself.

Comment (jeromelaban) [reply UnoCheck/Checkups/AndroidEmulatorCheckup.cs:123]: Thanks, that makes sense. Then let's explicitly look for arm64-v8a (the other arm emulators will likely run slowly anyways). With that change make sure to add in the code comment those sample strings for future refence.

Comment (pictos) [reply UnoCheck/Checkups/AndroidEmulatorCheckup.cs:123]: @jeromelaban this `arm64-v8a` should be a const? If so do we have a good place for it?

Comment (jeromelaban) [reply UnoCheck/Checkups/AndroidEmulatorCheckup.cs:123]: I don't know if there is, but you can at least make it a const at the top of this file.

**Review by jeromelaban — APPROVED** [review]

**Pull request merged**